src/vivarium_gates_nutrition_optimization/components/observers.py

```diff
@@ -95,16 +95,51 @@ def setup(self, builder: Builder) -> None:
             builder.results.register_observation(
                 name=f"anemia_{anemia_category}_person_time",
                 pop_filter=f'alive == "alive" and anemia_levels == "{anemia_category}" and tracked == True',
-                aggregator=self.aggregate_state_person_time,
+                aggregator=lambda df: len(df) * to_years(self.step_size()),
```

Comment:
You're using the same aggregator function in both this and the `MaternalBMIObserver`. Consider defining it somewhere both observers have access to it to avoid duplication

Reply:
done